main.nf

```diff
@@ -168,7 +168,7 @@ process normalise {
         path "${outfile_basename}", emit: final_output_vcf
 
     """
-    bcftools norm -c ws -f genome.fa -N variants_remapped_sorted.vcf.gz -o ${outfile_basename} -O v
+    bcftools norm -f genome.fa  variants_remapped_sorted.vcf.gz -o ${outfile_basename} -O v
```

Comment:
```suggestion
    bcftools norm --check-ref -f genome.fa  variants_remapped_sorted.vcf.gz -o ${outfile_basename} -O v
```

I'm not entirely sure about this suggestion, so this is more a point for discussion. My reasoning is:
* Because of the allele conversion algorithm, we expect that REFs will actually **always** align with the genome;
* If this is the case, it won't hurt to include `--check-ref` (equivalent to `-c e`; exit with an error if there is REF mismatch), as we expect it to do nothing;
* However, this can serve as an additional assert/safeguard that will fail loudly _in case_ something went wrong (e.g. due to some bug upstream)

Reply:
I actually had the exact opposite thought: We know it'll always be the same as the reference so no need to check but I guess your point stands.